content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx

```diff
@@ -321,6 +321,25 @@ const getActiveusers = await prisma.user.findMany({
 
 > **Note**: It is not currently possible to [return the count of a relation](https://github.com/prisma/prisma/issues/5079).
 
+### Sort by relevance (PostgreSQL)
+
+In [3.5.0](https://github.com/prisma/prisma/releases/3.5.0) and later, when using PostgresSQL you can sort records by relevance to the query. This uses the relevance ranking functions from PostgreSQL's full text search feature, which are explained further in [the PostgreSQL documentation](https://www.postgresql.org/docs/9.1/textsearch-controls.html).
+
+Ordering by relevance can be used either separately from or together with search: `_relevance` is used to order the list, while `search` filters the unordered list. For example, the following query uses `_relevance` to order the list of users by relevance to the search term `'developer'` in their bio, and `search` to filter the list:
```

Comment:
I would introduce the `_relevance` keyword before explaining your can use it separately or together with `search`.

Something like:
> You can sort results by relevance using the `_relevance` parameter in the `orderBy` section of a Prisma client query.
> It can be used either separately or together with the `search` parameter: ...